app/request.go

```diff
@@ -60,14 +85,108 @@ func NewRequestCommand(getRequestClientFn GetRequestClientFn) (CommandOut, error
 			Flags: []cli.Flag{
 				&cli.StringFlag{
 					Name:     "request-id",
-					Usage:    "The request-id of the asynchronous request",
+					Usage:    "The request-id of the request",
+					Aliases:  []string{"r"},
+					Required: true,
+				},
+			},
+			Action: func(ctx *cli.Context) error {
+				res, err := c.getRequestStatus(ctx, ctx.String("request-id"))
+				if err != nil {
+					return err
+				}
+				return PrintProto(res)
+			},
+		}, {
+			Name:    "wait",
+			Usage:   "wait for the request complete",
+			Aliases: []string{"w"},
+			Flags: []cli.Flag{
+				&cli.StringFlag{
```

Comment:
We could define this as a var since it is a repeated flag

Reply:
fixed.